manual_upload.md

```
@@ -0,0 +1,43 @@
+# Manual TestFlight Upload Instructions
+
+Since automated upload is having authentication issues, here are your options:
+
+## Option 1: Transporter App (Recommended)
+1. Download "Transporter" from Mac App Store (free Apple app)
+2. Open Transporter and sign in with Apple ID: proud@me.com
+3. Drag and drop the IPA file: `./builds/Nuri-TestFlight.ipa`
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)